_data/api/api_hubeau_surv_litt.md

```diff
@@ -26,8 +26,8 @@ keywords:
 themes:
   - Environnement
 contact_link: newshubeau@brgm.fr
-doc_tech_link: https://hubeau.eaufrance.fr/api/vbeta/surveillance_littoral/api-docs
-last_update: 16/04/2021
+doc_tech_link: https://hubeau.eaufrance.fr/api/v1/surveillance_littoral/api-docs
```

Comment:
This page `https://hubeau.eaufrance.fr/api/v1/surveillance_littoral/api-docs` is displaying an error 
<img width="1509" alt="Capture d’écran 2023-02-10 à 11 15 05" src="https://user-images.githubusercontent.com/43042737/218065711-0df6c25f-03d2-4172-b9f5-45e581072f23.png">

Reply:
Bonjour, cette page n'est pas destinée à être lue par un navigateur, mais à être intégrée dans un outil. 
Donc Ok pour nous.